src/sas/qtgui/MainWindow/PackageGatherer.py

```diff
@@ -146,7 +147,7 @@ def get_imported_packages(self):
             if hasattr(package, '__version__'):
                 # Module has __version__ attribute
                 try:
-                    package_versions_dict[package_name] = package.__version__
+                    package_versions_dict[package_name] = sas.system.version.__version__
```

Comment:
This change should be reverted. This was storing the version of each dependency.

Reply:
yeah, didn't spot that.